### PR TITLE
Improve the project README

### PR DESCRIPTION
**Pull request by Calinou**
- Use the SVG version of the logo.
- Mention console support in the first paragraph.
- Point contributors to the `#godotengine-devel` IRC channel.
- Improve formatting for better readability.
- Add more links when needed.
- Fix warnings reported by markdownlint.

[Formatted preview](https://github.com/Calinou/godot/tree/improve-readme#readme)

### CODE REVIEW

**Comment by pycbouh** [comment]
Is it just me, or does the first sentence miss a verb?

**Comment by Calinou** [comment]
@pycbouh Good catch. I tried to improve the first sentence; is it better now?

**Comment by pycbouh** [comment]
@Calinou You didn't push anything 🙂

**Comment by Calinou** [comment]
@pycbouh Fixed.

**Comment by pycbouh** [comment]
Yes, much better now.

**Review by clayjohn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
+    <img src="logo.svg" width="400" alt="Godot Engine logo">
+  </a>
+</p>
 
-Homepage: https://godotengine.org
+## 2D and 3D cross-platform game engine
 
-#### 2D and 3D cross-platform game engine
+**[Godot Engine](https://godotengine.org) is a feature-packed, cross-platform
+game engine that can be used to create 2D and 3D games from a unified
```
Comment (akien-mga) [inline README.md:12]: The addition of "that can be used" makes it a bit wordy IMO. I know @pycbouh raised the concern that the verb seemed missing, but AFAIK it wasn't incorrect before: "a hammer is a tool to put nails in surfaces" / "Godot is a game engine to create games".

File: README.md [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
+interface.** It provides a comprehensive set of common tools, so that users can
+focus on making games without having to reinvent the wheel. Games can be
+exported in one click to a number of platforms, including the major desktop
+platforms (Linux, macOS, Windows), mobile platforms (Android, iOS), as well as
+Web-based platforms (HTML5) and
+[consoles](https://docs.godotengine.org/en/latest/tutorials/platform/consoles.html)
```
Comment (akien-mga) [inline README.md:18]: This is Godot's official PR pitch and not just the GitHub description, so I'd tend to keep it general and not good too deep into details. So adding "and consoles" is good, but I'd leave out the "(via third-party providers)", and possibly the link.

File: README.md (hunk 1 above)
Comment (pycbouh) [reply README.md:12]: I didn't want to imply that it was incorrect, just that it didn't flow well when I was reading it. I think that additional qualifiers after "to create" make it much weirder to read than "a tool to put nails".

File: README.md (hunk 2 above)
Comment (Calinou) [reply README.md:18]: I removed `(via third-party providers)` but left the link in, as the fact that console support is only available via third-party providers shouldn't be overlooked.

File: README.md (hunk 1 above)
Comment (akien-mga) [reply README.md]: I agree that the sentence is a bit heavy on qualifiers which make it a bit wordy. I wasn't convinced by adding "that can be used", but I'm open to other suggestions to improve this sentence so that it reads better.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.3.
